stage1/init/init.go

```diff
@@ -279,7 +279,7 @@ func getArgsEnv(p *stage1commontypes.Pod, flavor string, canMachinedRegister boo
 		// kernel and hypervisor binaries are located relative to the working directory
 		// of init (/var/lib/rkt/..../uuid)
 		// TODO: move to path.go
-		kernelPath := filepath.Join(common.Stage1RootfsPath(p.Root), "bzImage")
+		kernelPath := filepath.Join(common.Stage1RootfsPath(p.Root), "kernel_image")
```

Comment:
I see `bzImage` was explicitely targeted in make invocation before. I am not too much concerned with this rename, but is it being for some specific purpose or just for cosmetic reasons?